pandas/tests/internals/test_internals.py

```diff
@@ -297,7 +297,8 @@ def test_delete(self):
         assert (newb.values[1] == 1).all()
 
         newb = self.fblock.copy()
-        with pytest.raises(Exception):
+
+        with pytest.raises(IndexError, match=None):
```

Comment:
What is the `None` here for? Can this not be made more specific?

Reply:
This can be more specific as soon as https://github.com/pandas-dev/pandas/pull/31130 gets merged.